docs/embedded/concepts/spe-copilot.md

```diff
@@ -0,0 +1,262 @@
+
+### Retrevial API
+
+### M365 Boundrary 
+
+Data is kept secure: data never leaves the tenant boundary and storage respects data residency settings​.
+
+Each container instance of a container type in the SPE parition is its own security and complaince boundrary.
+
+![M365 Storage Partitions](../images/speco-bound.png)
+
+### CSP Policies
+
+ The Content-Security-Policy (CSP) for embedded chat hosts, ensures that only specified hosts can load the `chatembedded.aspx` page. This helps in securing the application by restricting which domains can embed the chat component. 
+
+ It is intended to allow consuming tenant SPE admins to set an allowlist of hosts that they will allow to embed the SPE copilot in an iframe. Specifically, the value they set here will be used in a Content-Security-Policy header as a frame-ancestors value. 
+
+> [!NOTE]
+>
+> If this configuration is not set, the [Content-Security-Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy) will default be set to
+> [frame-ancestors](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/frame-ancestors): ‘none’ which means no one can embed the copilot. 
+
+Below are example commands to use the [SPO Admin Powershell](https://learn.microsoft.com/en-us/powershell/module/sharepoint-online/set-spocontainertypeconfiguration?view=sharepoint-ps) to set teh `CopilotEmbeddedChatHosts` property.
+
+```powershell
+Set-SPOContainerTypeConfiguration -ContainerTypeId 4f0af585-8dcc-0000-223d-661eb2c604e4 -CopilotEmbeddedChatHosts "http://localhost:3000 https://contoso.com https://fabrikam.contoso.com" 
+
+# This should set the container type configuration “CopilotEmbeddedChatHosts” accordingly. 
+...
+
+Get-SPOContainerTypeConfiguration -ContainerTypeId 4f0af585-8dcc-0000-223d-661eb2c604e4 
+
+# This should return the container type configurations and their values including CopilotEmbeddedChatHosts 
+```
+
+### Consumptive Billing ( Coming Soon )
+
+### Supported Document Types
+
+[Reference - File Formats Support By Copilot](https://support.microsoft.com/en-us/topic/file-formats-supported-by-copilot-1afb9a70-2232-4753-85c2-602c422af3a8)
+
+<b>Documents</b>: PDF, DOCX, XLSX, PPTX
+
+<b>Text-based Files</b>: RTF, TXT, CSV, LOG, INI, CONFIG
+
+<b>Audio</b>: WAV
+
+<b>Programming Languages</b>: PY, JS, JSX, JAVA, PHP, CS, C, CPP, CXX, H, HPP, M, COFFEE, DART, LUA, PL, PM, RB, RS, SWIFT, GO, KT, KTS, R, SCALA, T, TS, TSX
+
+<b>Shell Scripts</b>: BASH, SH, ZSH
+
+<b>Markup and Documentation</b>: HTML, CSS, MD, RMD, TEX, LATEX
+
+<b>Database Languages</b>: SQL
+
+<b>Data Serialization Formats</b>: IPYNB, JSON, TOML, YAML, YML
+
+
+## Getting Started
+
+
+> [!NOTE]
+>
+> 1. You must specify the purpose of the container type you're creating at creation time. Depending on the purpose, you may or may not need to provide your Azure Subscription ID. A container type set for trial purposes can't be converted for production; or vice versa.

```

Comment:
This is only available with standard container type so make it clearer here.

Move all the other sections related to usage under this Getting started section